pkg/bindings/connection.go

```diff
@@ -149,23 +152,83 @@ func sshClient(_url *url.URL, uri string, identity string, machine bool) (Connec
 	connection := Connection{
 		URI: _url,
 	}
+	userinfo := _url.User
+	if _url.User == nil {
+		u, err := user.Current()
+		if err != nil {
+			return connection, fmt.Errorf("current user could not be determined: %w", err)
+		}
+		userinfo = url.User(u.Username)
+	}
 	port := 22
 	if _url.Port() != "" {
 		port, err = strconv.Atoi(_url.Port())
 		if err != nil {
 			return connection, err
 		}
 	}
+	// ssh_config
+	alias := _url.Hostname()
+	match := false
+	cfg := ssh_config.DefaultUserSettings
+	if val := cfg.Get(alias, "User"); val != "" {
+		userinfo = url.User(val)
+		match = true
+	}
+	if val := cfg.Get(alias, "Hostname"); val != "" {
+		uri = val
+		match = true
+	}
+	if val := cfg.Get(alias, "Port"); val != "" {
+		if val != ssh_config.Default("Port") {
+			port, err = strconv.Atoi(val)
+			if err != nil {
+				return connection, fmt.Errorf("port is not an int: %s: %w", val, err)
+			}
+			match = true
+		}
+	}
+	if val := cfg.Get(alias, "IdentityFile"); val != "" {
+		if val != ssh_config.Default("IdentityFile") {
+			identity = strings.Trim(val, "\"")
+			match = true
+		}
+	}
+	if match {
+		logrus.Debugf("Alias: %s", alias)
+		logrus.Debugf("  User: %s", userinfo)
+		logrus.Debugf("  Hostname: %s", uri)
+		logrus.Debugf("  Port: %d", port)
+		logrus.Debugf("  IdentityFile: %s", identity)
```

Comment:
I think it should add some form of `SSH config ...` prefix to each message so it is clear what exactly is being printed. OT I guess on the first Alias message it would be enough.

Reply:
OK, will try to come up with something longer. Perhaps user/hostname/port can go on one line (host), as well.

i.e. like CONTAINER_HOST and CONTAINER_SSHKEY environment variables